backend/tests/integration/records/api/test_api_literature.py

```diff
@@ -1781,3 +1781,218 @@ def test_adding_files_with_s3_url_but_wrong_key(
 
         assert record["figures"] == expected_figures
         assert record["documents"] == expected_documents
+
+
+def test_creating_record_updates_entries_in_authors_records_table(
+    base_app, db, create_record
+):
+    expected_authors_entries_count = 5
+    expected_table_entries = [
+        "K.Janeway.1;INSPIRE BAI",
+        "INSPIRE-12345678;INSPIRE ID",
+        "J.T.Kirk.1;INSPIRE BAI",
+        "collaboration_1;collaboration",
+        "collaboration_2;collaboration",
```

Comment:
this is morally a set of tuples, why not make it effectively a set of tuples?

Reply:
:+1: